### PR TITLE
API-7535 reorg oauth proxy

### PR DESCRIPTION
**Pull request by crolarlibertyva**
Move contents of oauth-proxy to its parent folder

### CODE REVIEW

**Comment by crolarlibertyva** [comment]
Must be coordinated with https://github.com/department-of-veterans-affairs/devops/pull/9387

File: tests/bats/regression_tests.sh [hunk 1]
```diff
@@ -112,7 +112,7 @@ assign_code() {
   local network=""
   if [[ $HOST == *"localhost"* ]];
   then
-    network="-i --network container:oauth-proxy_oauth-proxy_1"
+    network="-i --network container:lighthouse-oauth-proxy_oauth-proxy_1"
```
Comment (beaugrantham-va) [inline tests/bats/regression_tests.sh:115]: I'm not sure this should actually change? It's the container name which is based on the service name in docker-compose... which hasn't changed.

Comment (beaugrantham-va) [reply tests/bats/regression_tests.sh:115]: Disregard, it appears to be correct.

**Review by lukemarkwordtlibertyits — APPROVED** [review]
Sorry that I just merged a change to the buildspec-deploy. Other than the conflict this looks good.

**Pull request merged**